tests/roots/test-search/index.rst

```diff
@@ -0,0 +1,8 @@
+meta keywords
+=============
+
+.. meta::
+	:keywords lang=en: findthiskey
+	:keywords lang=de: findnotthiskey
+	:keywords: thisonetoo
```

Comment:
Could you add non-keywords metadata (like descpription)?
And we should check search indecies does not contain them.